.github/workflows/build_and_deploy.yml

```
@@ -241,12 +241,18 @@ jobs:
           ESIGNERCKA_USERNAME: ${{ secrets.ESIGNERCKA_USERNAME }}
           ESIGNERCKA_PASSWORD: ${{ secrets.ESIGNERCKA_PASSWORD }}
           ESIGNERCKA_TOTP_SECRET: ${{ secrets.ESIGNERCKA_TOTP_SECRET }}
-      - name: Upload artifact to build XCFramework
+      - name: Upload artifact to build XCFramework(iOS)
         if: matrix.c_release_format == 'ios-xcframework'
         uses: actions/upload-artifact@v4
         with:
           name: voicevox_core-${{ matrix.target }}
           path: artifact/${{ env.ASSET_NAME }}
+      - name: Upload artifact to build XCFramework(macOS)
+        if: matrix.artifact_name == 'osx-arm64' || matrix.artifact_name == 'osx-x64'
+        uses: actions/upload-artifact@v4
+        with:
+          name: voicevox_core-${{ matrix.target }}
+          path: artifact/${{ env.ASSET_NAME }}
```

（細かいけど）
この２つ、目的が一緒で処理も一緒だからくっつけても良いかも･･･？

```suggestion
      - name: Upload artifact to build XCFramework
        if: matrix.artifact_name == 'osx-arm64' || matrix.artifact_name == 'osx-x64' || matrix.c_release_format == 'ios-xcframework'
        uses: actions/upload-artifact@v4
        with:
          name: voicevox_core-${{ matrix.target }}
          path: artifact/${{ env.ASSET_NAME }}
```